frontend/src/components/review/ExistingComment.tsx

```diff
@@ -118,12 +118,12 @@ const ExistingComment = ({
         {content}
       </Text>
       <Flex>
-        {!commentIndex && (
+        {!resolved && !commentIndex && (
```

Comment:
fyi theres brain boggling ticket coming up for comment ux. It'll be simpler to keep these conditions for resolving + replying separately